### DISCUSSION TITLE
Assistance with building a Docker image

### DISCUSSION THREAD

**Discussion by mrfancyfrancis** [OwlGamingCommunity/V#17] [Support] [answered]
Hi. I'm currently trying to initialize (and eventually code) using this as a base. I'm on a non-Windows computer so I've been trying to make Docker work.

I've tried running docker build on the repository (fresh pull and copypasted the ragemp-server files on Output) and I've gotten this error every time.
![image](https://user-images.githubusercontent.com/19979112/133310292-ba6c1310-12d9-4008-9173-152a71886c6b.png)

I've copypasted a copy of the ragemp server onto Output/Debug/
![image](https://user-images.githubusercontent.com/19979112/133310618-8673fbc9-05eb-4068-ab92-2b69d1a1f4f0.png)

Anything I should do/tweak or anything I've missed out before building an image? For reference, the released package works perfectly when I ran it.

Any response is appreciated. Thank you.

**Comment by ChaosParadise** [comment, marked as answer]
To me this looks like you're not cloning using git LFS so you are missing contents in your DLL files. Check the file size, they should not be 1KB.

**Reply by mrfancyfrancis** [reply]
sorry for the late response but I already solved it and LFS was the answer.

Curiously though, I tried downloading it as a ZIP file (so not thru git cloning) and it still gave me an error. Any insight as to why?

**Reply by ChaosParadise** [reply]
Because downloading the ZIP doesn't include Git LFS objects.
